### PR TITLE
Autoloader and autoejecter stop rollers when terminated

### PR DESCRIPTION
**Pull request by AidanSmith**
Rollers should now stop when the autoloader ends.

### CODE REVIEW

File: QuasarHelios2015/src/org/team1540/quasarhelios/AutoEjector.java [hunk 1]
```diff
@@ -37,14 +37,22 @@ protected void autonomousMain() throws AutonomousModeOverException, InterruptedE
...
-            Rollers.direction.set(true);
-            Rollers.running.set(true);
-
-            waitUntil(BooleanMixing.invert(AutoLoader.crateInPosition));
-            waitForTime(timeout);
-
-            Rollers.running.set(false);
+            boolean r = Rollers.running.get();
+            boolean d = Rollers.direction.get();
+            boolean o = Rollers.closed.get();
```
Comment (vincent-miller) [inline QuasarHelios2015/src/org/team1540/quasarhelios/AutoEjector.java]: It's probably best to have more descriptive names for these. "running," "direction," and "open" would be fine.

**Comment by vincent-miller** [comment]
Just one small thing

File: QuasarHelios2015/src/org/team1540/quasarhelios/AutoEjector.java [hunk 2]
```diff
@@ -37,14 +37,22 @@ protected void autonomousMain() throws AutonomousModeOverException, InterruptedE
...
+
+            try{
+                Rollers.closed.set(true);
+                Rollers.direction.set(true);
+                Rollers.running.set(true);
+
+                waitUntil(BooleanMixing.invert(AutoLoader.crateInPosition));
```
Comment (celskeggs) [inline QuasarHelios2015/src/org/team1540/quasarhelios/AutoEjector.java:53]: Also, `waitUntilNot` exists now, so we can probably use it.

**Comment by celskeggs** [comment]
Just one thing from me.

**Comment by AidanSmith** [comment]
Changed it.

**Comment by amackworth** [comment]
There are still some conflicts. Try again?

**Pull request merged**